### DISCUSSION TITLE
友链 | YuZhangWang的领域

### DISCUSSION THREAD

**Discussion by giscus** [YuZhangWang/YuZhangWang.github.io#11] [Announcements]
# 友链 | YuZhangWang的领域

> 每次刷新为随机顺序展示～
> 现已不支持在评论区中直接申请友链，请在[Blog-Friend-Link](https://github.com/YuZhangWang/Blog-Friend-Link)这个 Repo 里按要求提交一个 Pull Request 就可以申请友链啦~

https://www.yuzhang.wang/link/

**Comment by tzy13755126023** [comment]
name: Ethan.Tzy
link: https://tzy1997.com/
avatar: https://bu.dusays.com/2022/05/02/626f92e193879.jpg
descr: 古今之成大事者，不惟有超世之才，亦必有坚忍不拔之志
screenshot: https://bu.dusays.com/2022/06/01/6296ceb2e4935.jpg

**Reply by YuZhangWang** [reply]
感谢留言，已添加！

**Comment by Achuan-2** [comment]
大佬好，看到你在我博客的留言了，非常感谢！！！
但是我现在已经不折腾hexo了，把本地的hexo删了，奔向语雀的怀抱了（大概人老了，就不想折腾了吧）
所以就加不了你友链了。。。
只好把你博客加入书签了，有时间会来踩踩的！

**Reply by YuZhangWang** [reply]
好，麻烦了，谢谢收藏

**Reply by YuZhangWang** [reply]
大佬，我看你加入了思源笔记的组织。我发现里面的成员，D和V，我和他们有过接触，和他们也交换过友链，不知道大佬能否邀请我加入下组织呢？

**Reply by Achuan-2** [reply]
> 大佬，我看你加入了思源笔记的组织。我发现里面的成员，D和V，我和他们有过接触，和他们也交换过友链，不知道大佬能否邀请我加入下组织呢？

这个你自己联系D吧，他邮箱是845765@qq.com，我是之前有制作社区主题才加入组织的

**Reply by YuZhangWang** [reply]
> > 大佬，我看你加入了思源笔记的组织。我发现里面的成员，D和V，我和他们有过接触，和他们也交换过友链，不知道大佬能否邀请我加入下组织呢？
> 
> 这个你自己联系D吧，他邮箱是[845765@qq.com](mailto:845765@qq.com)，我是之前有制作社区主题才加入组织的



好的，已经联系了，D给我发了邀请，刚刚加入了

**Comment by miaoermua** [comment]
请修改一下本站信息

**Reply by miaoermua** [reply]
本站信息
站点标题: 喵二の小博客
站点描述: 缘，妙不可言
主人头像: https://www.miaoer.xyz/weblogo.jpg

**Reply by YuZhangWang** [reply]
抱歉回复晚了，已经修改好了

**Reply by YuZhangWang** [reply]
请更新一下我得头像链接，谢谢
> https://gcore.jsdelivr.net/gh/YuZhangWang/Creative-pictures02@master/img/202210171416164.png

**Reply by miaoermua** [reply]
> 请更新一下我得头像链接，谢谢
> > https://gcore.jsdelivr.net/gh/YuZhangWang/Creative-pictures02@master/img/202210171416164.png

好的

**Comment by Ran-Xing** [comment]
name: XR-SEC
link: https://xrsec.eu.org
avatar: https://xrsec.s3.bitiful.net/IMG/20201208143114085332.png?fmt=webp&q=48
descr: 低调求发展，潜心习安全
麻烦更改下

**Reply by YuZhangWang** [reply]
收到，已经修改好了

**Reply by YuZhangWang** [reply]
感谢留言告知

**Comment by tzy13755126023** [comment]
name: Ethan.Tzy
link: https://fe32.top/

域名已经换成 fe32.top 了， 麻烦贵站修改下 友链信息 ，谢谢~

**Reply by YuZhangWang** [reply]
OK

**Reply by YuZhangWang** [reply]
我的网站最近恢复了，麻烦把我从失联中移出来吧

**Reply by YuZhangWang** [reply]
麻烦更新一下我的网站描述：PLUSULTRA!

**Comment by imjiaoyuan** [comment]
已经添加贵站，劳烦更新

name: Hieroglyphs
link: https://hieroglyphs.top
avatar: https://hieroglyphs.top/images/avatar.png
descr: 世不患无才，患无用之之道
siteshot: https://hieroglyphs.top/images/siteshot.png

**Reply by YuZhangWang** [reply]
你好，感谢你的互链，关于我的友链添加的具体操作，请根据[如何交换友链](https://github.com/YuZhangWang/Blog-Friend-Link#%E5%A6%82%E4%BD%95%E4%BA%A4%E6%8D%A2%E5%8F%8B%E9%93%BE)中的步骤进行pr操作，即可在我的网站上进行添加友链

**Reply by YuZhangWang** [reply]
抱歉，我并没有在你的友链页面看到我的链接，请添加之后，再根据[如何交换友链](https://github.com/YuZhangWang/Blog-Friend-Link#%E5%A6%82%E4%BD%95%E4%BA%A4%E6%8D%A2%E5%8F%8B%E9%93%BE)中的步骤进行pr操作

**Comment by hjljy** [comment]
海加尔金鹰的友链信息麻烦更新下

博客名:弋钓草野
博客地址:https://www.aliuying.com
博客图片地址:https://www.aliuying.com/favicon.png
博客介绍:如野草般一岁一枯荣
联系方式：hjljy@outlook.com

**Reply by YuZhangWang** [reply]
请移步[https://github.com/YuZhangWang/Blog-Friend-Link](https://github.com/YuZhangWang/Blog-Friend-Link)，按照[格式](https://github.com/YuZhangWang/Blog-Friend-Link#%E6%A0%BC%E5%BC%8F)和[如何交换友链](https://github.com/YuZhangWang/Blog-Friend-Link#%E5%A6%82%E4%BD%95%E4%BA%A4%E6%8D%A2%E5%8F%8B%E9%93%BE)进行操作，请仔细跟着步骤来进行Pr提交。因为你的友链本身就有了，所以你只需要把你的部分修改一下就可以了

**Reply by YuZhangWang** [reply]
此外，请修改一下我的网站描述，修改为   PLUSULTRA!

**Reply by YuZhangWang** [reply]
Update，不用麻烦了，我已经修改完成了

**Reply by hjljy** [reply]
网站描述更新了，但是我看你连接和博客名更新失败了

**Reply by YuZhangWang** [reply]
> 网站描述更新了，但是我看你连接和博客名更新失败了

是指我的网站还是你的网站
我的网站的话，你是不是没用科学上网？我没有做国内加速
你的网站的话。我刚刚看了下，链接是修改了，博客名字似乎没有改，你可以提交一个PR修改一下

**Comment by hjljy** [comment]
很久没有科学上网了

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2023年10月12日 13:07
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [YuZhangWang/YuZhangWang.github.io] 友链 | YuZhangWang的领域 (Discussion #11)


网站描述更新了，但是我看你连接和博客名更新失败了

是不是没用科学上网？我没有做国内加速

―

**Reply by YuZhangWang** [reply]
国内加速要备案，备案审查标准又更新过，对于一些内容不太方便，干脆不做加速了

**Comment by hjljy** [comment]
是的，备案挺麻烦的，还好我这个域名很早之前就备案了

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2023年10月12日 13:45
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [YuZhangWang/YuZhangWang.github.io] 友链 | YuZhangWang的领域 (Discussion #11)


国内加速要备案，备案审查标准又更新过，对于一些内容不太方便，干脆不做加速了

―

**Reply by YuZhangWang** [reply]
我之前也备案过，后来过了一年就被取消掉了，至今也没有查明原因，我也就懒得备案了

**Comment by ghser** [comment]
哈喽，“一叶三秋”现在改成“秋日续”咯
```yml
- site: 秋日续📸
  url: https://upchiu.com
  desc: 用镜头记录美好生活📷
  image: https://cdn.upchiu.com/2023/10/20231020111406671-150x150.jpg
```

**Comment by ghser** [comment]
哈喽，“一叶三秋”现在改成“秋日续”咯
```yml
- site: 秋日续📸
  url: https://upchiu.com
  desc: 用镜头记录美好生活📷
  image: https://cdn.upchiu.com/2023/10/20231020111406671-150x150.jpg
```

**Reply by YuZhangWang** [reply]
Please see [注意事项](https://yuzhang.wang/link/#%E6%B3%A8%E6%84%8F%E4%BA%8B%E9%A1%B9)

**Comment by baozongwi** [comment]
师傅我想问一下，这种每次刷新友链顺序就会不一样的操作怎么弄的

**Reply by YuZhangWang** [reply]
这个的话，我是用Hexo框架来搭建的，Hexo自带的功能，配置里有。

**Comment by miaoermua** [comment]
你好请将我站链接更新一下，已经发了好几封邮件提醒了，如果该邮箱不使用请与我取得联系。

- https://www.miaoer.net
- https://static.miaoer.net/logo/avatar.webp

**Reply by YuZhangWang** [reply]
现已不支持在评论区中直接申请友链，请在[Blog-Friend-Link](https://github.com/YuZhangWang/Blog-Friend-Link)这个 Repo 里按要求提交一个 Pull Request 就可以申请友链啦~

**Reply by YuZhangWang** [reply]
这个公告已经发布1年有余了，并且就在评论区上方
